### DISCUSSION TITLE
Cannot edit in wiki

### DISCUSSION THREAD

**Discussion by joemarker2** [community/community#38831] [New to GitHub]
I am new to gitHub.  I am trying to add new content under wiki.  I am told that there is supposed to be an "Edit" button in the upper right hand corner.  I do not see such a button.  Do you have any ideas about what I need to do?

**Comment by airtower-luna** [comment]
Do you have write access to the repository? Repository owners can restrict access to collaborators.

**Reply by joemarker2** [reply]
I am able to add issues.  Does that mean I have write access?

**Reply by airtower-luna** [reply]
No, that doesn't say anything. Can you commit directly to the repository (not a fork)?

**Reply by gitshiven** [reply]
Yes, when you go the file inside repository you will find an edit button somewhere on the right side.
![Web capture_12-11-2022_0215_github com](https://user-images.githubusercontent.com/91045496/201407970-1f648ee1-163b-4f70-82ed-77dae803aacb.jpeg)

**Reply by airtower-luna** [reply]
That doesn't say anything, if you don't have direct write access pressing that button [will create a fork so you can edit there](https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files#editing-files-in-another-users-repository) and make a PR with your change.

**Reply by druskisa** [reply]
how do i go the file inside repository?

**Comment by jfullstackdev** [comment]
if you have the screenshot of that page, then it's better so we can take a look

for example, if I visited a random repo, 

![image](https://user-images.githubusercontent.com/47092464/201454740-0e1501b5-bb00-4b93-b5b2-7f5efbaab0e9.png)

that is my first view, no edit button at all, if I click the exact file, even if I have no Write Access, there is the Edit button

![image](https://user-images.githubusercontent.com/47092464/201454760-0502d2f7-0a17-4422-b023-6c6c5ae58cfb.png)

but, you can't edit that directly, you will be editing just the copy of that if you are not part of them
